media-libs/libhdhomerun/libhdhomerun-20190621.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SiliconDust HDHomeRun Utilties"
+HOMEPAGE="https://www.silicondust.com/support/linux/"
+SRC_URI="https://download.silicondust.com/hdhomerun/${PN}_${PV}.tgz"
+
+PATCHES=(
```

Comment:
Generic suggestion is to keep PATCHES below the basic variables, i.e. below where  you keep RDEPEND right now.

Reply:
Moved PATCHES